content/en/news/releases/1.12.x/announcing-1.12/upgrade-notes/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.12.0.
+weight: 20
```

Comment:
This missing a date leaves it out of the News section of the website
<img width="968" alt="Screen Shot 2021-11-18 at 11 57 47 AM" src="https://user-images.githubusercontent.com/10537847/142470657-0c47cae3-4e70-4889-a924-5b73baefa699.png">
e.
 vs 
<img width="1004" alt="Screen Shot 2021-11-18 at 11 58 22 AM" src="https://user-images.githubusercontent.com/10537847/142470727-f6447eef-aa5a-46ad-9529-5d846f5b9011.png">